generate.go

```diff
@@ -9,9 +9,8 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/Sirupsen/logrus"
+	"github.com/opencontainers/ocitools/libgen"
```

Comment:
@wking , @mrunalp , I think I will first get myself familiar with #54 , and #128 . 
Then we can come back to this PR and decide how we want it to be.